src/app/mi-details/mi-details.page.html

```diff
@@ -0,0 +1,90 @@
+<app-application-header [config]="headerConfig" (actionClick)="handleSaveClick($event)"></app-application-header>
+<ion-content class="ion-padding">
+  <div class="spacing ion-padding">
+    <h1 class="text-header">{{"GENERATED_AND_EDITED_TITLE" | translate}}</h1>
+    <div class="text-muted">
+      <span>{{"BY" | translate}} </span>
+      <div class="author-contianer">
+        @if(details.author){
+          <span>{{details.author}}</span>
+        }
+        @if(details.location){
+          <span>{{details.location}}</span>
+        }
+      </div>
+    </div>
+  </div>
+  @if(details.title){
+    <div class="card">
+      <h1 class="card-title">
+        <ion-icon name="star"></ion-icon>
+        {{"PROBLEM_STATEMENT" | translate}}
+      </h1>
+      <p class="card-text">
+        {{details.title}}
+      </p>
+    </div>
+  }
+
+@if(details.description){
+  <div class="card">
+    <h1 class="card-title">
+      <ion-icon name="star"></ion-icon>
+      {{"OBJECTIVE" | translate}}
+    </h1>
+    <p class="card-text">
+      {{details.description}}
+    </p>
+  </div>
+}
+
+ @if(details.tasks){
+  <div class="card">
+    <h1 class="card-title">
+      <ion-icon name="star"></ion-icon>
+      {{"ACTION_STEPS" | translate}}
+    </h1>
+    <ul class="numbered-list">
+      @for(task of details.tasks; track task){
+        <li>{{task.name}}</li>
+      }
+    </ul>
+  </div>
+ }
+
+@if(details.learningResources){
+  <div class="card">
+    <h1 class="card-title">
+      <ion-icon name="star"></ion-icon>
+      {{"IMPACT" | translate}}
+    </h1>
+    <p class="card-text">
+      {{details.learningResources[0].name}}
+    </p>
+  </div>
```

Comment:
the structure is re-usable can you use a ng-template to re-use the structure and pass the dynamic data to it?